upload_log_files.py

```diff
@@ -0,0 +1,77 @@
+import argparse
```

Comment:
Can you add a run script for this? I think it would be make sense and be helpful to have a script that configures this to run either on a schedule or at the end of a pipeline run.

Reply:
added thanks.

Reply:
added thanks